src/app/qgsdxfexportdialog.cpp

```diff
@@ -43,6 +43,17 @@ QWidget *FieldSelectorDelegate::createEditor( QWidget *parent, const QStyleOptio
 {
   Q_UNUSED( option )
 
+  if ( index.column() == 2 )
```

Comment:
could we use named constants instead of those 0, 1, 2, 3 column indices ? (here and in many similar below instances) This would help a lot for code understanding